.buildkite/pipeline.yml

```diff
@@ -282,6 +282,19 @@ steps:
         artifact_paths: "held_suarez_equil/output_active/*"
         agents:
           slurm_constraint: icelake|cascadelake|skylake|epyc
+      
```

Comment:
We already have `gpu_aquaplanet_dyamond_summer` in ci. Let's not add more tests. Could you just change it to use ed only? Thanks!

Reply:
feel free to move it to model_configs too if you think that's better.